### ISSUE TITLE
unit correction in PlexonIO

### ISSUE THREAD

**Issue by tlmassey** [NeuralEnsemble/python-neo#372]
Hi, I've noticed a couple of unit errors in PlexonIO.  

1) It reads in the voltages of analog signals as volts rather than millivolts.  My actual signal levels in this example are tens of microvolts.

```
>>> activeseg.analogsignals[0][0]
array([-0.025]) * V
```

2) It reads in the sampling frequency of spike train waveforms as Hz rather than kHz.  My actual sampling frequency in this example is 1 kHz.

```
>>> activeseg.spiketrains[0].sampling_rate
array(1.0) * Hz
```

These are the only ones I've seen so far.  I'll post here if I catch any more.

Thanks for working so hard to make Neo and PlexonIO a great resource!

**Comment by apdavison** [comment]
@samuelgarcia could you take a look at this?

**Comment by samuelgarcia** [comment]
Yes I will.
I will try to fix several IO issue and many test that are no passing in the master in one commit.

For the milestone, we haven't discusss it yet but I was thinking that 0.6 could introduce the new neo.rawio. And so maybe 0.5.2 could be the bugfix and new IOs. So I would put  0.5.2 here. No ?

**Comment by apdavison** [comment]
Yes, 0.5.2 would be great

**Comment by samuelgarcia** [comment]
@tlmassey: could you send me the file you have please because on files  hosted atgnode for testing the SpikeTrain.sampling_rate seems to be OK.

**Comment by tlmassey** [comment]
Sure.  File is attached.

Screenshot of discrepancy between spike train sampling rate and analog
signal sampling rate (should be same, obviously).

[image: Inline image 1]

**Comment by samuelgarcia** [comment]
Unfortunatly when reply to github message, attached files disapear.
Could you send me the file directly yo my mail please ?
sam.garcia.die@gmail.com

**Comment by samuelgarcia** [comment]
PR #385 should fix units for Volts in AnalogSignal.
SpikeTrain.waveforms need to be resolved.

**Comment by apdavison** [comment]
Fixed in #385
